docs/docs/start-here.md

```diff
@@ -38,53 +42,58 @@ questions, which you can look for in the [Nullness Design FAQ]. If you like,
 
 ### Begin with one of
 
-* The current [User Guide] draft
-* The [Javadoc], which is not a *tutorial* walkthrough, but is thorough,
-  specific, and current.
+*   The [User Guide].
+*   The [Javadoc], which is not a *tutorial* walkthrough, but is thorough,
+    specific, and current.
 
 ### Then if you're *really* interested
 
-* The [specification], written to be understood by owners of compilers and
-  static analysis tools. Note that it currently matches JSpecify 0.2, not 0.3.
-* Our [wiki] has about 20 informal, non-normative articles on various topics
-* Open [issues]
-* Or try it out...
+*   The [specification], written to be understood by owners of compilers and
+    static analysis tools. Note that it currently matches JSpecify 0.2, not 0.3.
```

Comment:
Maybe change "0.3" to "1.0" here—or "and it will require some updates for 1.0" to make it sound less like we have already made every last decision? Or you could rephrase an even larger part of the sentence to just say more generally that it's out of date.